core/src/abi.rs

```diff
@@ -156,6 +158,17 @@ pub(crate) mod host_queries {
         meta_data(Metadata::BLOCK_HEIGHT).unwrap()
     }
 
+    /// Get the moonlight sender of the ongoing tx (if any).
```

Comment:
```suggestion
    /// Get the public sender of the ongoing tx. Returns `None` if the transaction is shielded.
```

Reply:
Agree
Additionally I'll rename moonlight_sender to public_sender as soon as I rebase this PR

Reply:
Cool, I like this way of thinking 🙂